portfolio/src/main/webapp/style.css

```diff
@@ -55,20 +91,142 @@ body {
     width: auto;
 }
 
+/*************** loader ***************/
+.loader {
+  border: 16px solid #ffffff; /* white */
+  border-top: 16px solid #3498db; /* Blue */
+  border-radius: 50%;
+  width: 120px;
+  height: 120px;
+  animation: spin 2s linear infinite;
+  
+  position: absolute;
+  top: 40%;
+  left: 50%;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+
+
+
+/*************** content ***************/
+
+#contentspacer{
+    height: 61px; /*61 to account for the 1px border on the 60px header*/
+}
+
+#horizontal-spacer{
+    width: 10%
+}
+
+#welcome-wrapper{
+    text-align:center;
+    display: inline-block;
+    width: 100%;
+}
+#welcome{
+    margin-top: 15%;
+    margin-bottom: 30%;
+}
+
+.websiteicons{
+    height: 80px;
+    width: auto;
+    padding: 10px;
+}
+#openertable{
+    display: inline-block;
+}
+
+/* this portion handles the hovering */
+#opener1{
+    font-size: 60px;
+}
+#opener2{
+    margin:auto;
+    font-size: 60px;
+    display: none;
+}
+
+div#welcome h1#opener1{
+    display: block;
+}
+div#welcome:hover h1#opener1{
+    display: none;
+}
+
+div#welcome h1#opener2{
+    display: none;
+}
+
+div#welcome:hover h1#opener2{
+    display: block;
+}
+
+div#welcome h2#computabeast{
+    font-size: 30px;
+    width: 30%;
+    padding: 10px;
+    background-color: #ccff00; /*highlighter*/
+    border-radius: 3px;
+    display:none;
+}
+
+div#welcome:hover h2#computabeast{
+    display:inline-block;
+}
+
+#opener2{
+    display: none;
+}
+
+/* end hovering portion */
 
-/* content */
 #content {
-    margin-top: 60px; /* to account for the fixed header */
-    margin-left: 10%;
+    margin-left: 5%;
     margin-right: auto;
-    width: 20%;
+    width: 90%;
+}
+
+#pictureTable {
+    flex: 1;
+}
+#tabler{
+    margin-top: 30px; /*had to do this bc margin-top wasn't working in #picturetable*/
+}
+
+.part{
+    display: flex;
+    margin-top: 5%;
 }
 
-#greeting-container {
-    margin-top: 20px;
+.text-container {
+    margin-right: 2%;
+    margin-left: 2%;
 }
 
-.contentimages{
-    height: 200px;
+.tablepics{
+    height: 20%; 
     width: auto;
+    transition: 1s ease-in-out; /* lazy -- will add other browsers later */
+}
+
+.tablepics.rightpics:hover{
+    transform: translate(-15em, 0) scale(2); /* lazy -- will add other browsers later */
```

Comment:
Keep non-html files at 80 characters per line. html files can be 100 or 120 characters depending on your conventions.